public/index.html

```diff
@@ -1,43 +1,20 @@
 <!DOCTYPE html>
```

Comment:
Please use Prettier to reformat this file

Reply:
I'm trying to reformat, my save removes the reformatting for some reason. I'm going to look at removing auto formatitng